force-app/main/default/classes/IBMNaturalLanguageClassifierV1Test.cls

```diff
@@ -0,0 +1,171 @@
+    System.assertEquals(ClassifiedClassList.size(), 2);
+    IBMNaturalLanguageClassifierV1Models.ClassifiedClass ClassifiedClass = ClassifiedClassList[0];
+    String class_name_serialized_name = ClassifiedClass.getClassName();
+    Set<String> class_name_set = new Set<String>{'temperature', 'conditions'};
+    System.assert(class_name_set.contains(class_name_serialized_name), 'Invalid class name for classifier.');
+    Double confidence_serialized_name = ClassifiedClass.getConfidence();
+    System.assertNotEquals(confidence_serialized_name, null);
+    System.assert(confidence_serialized_name>0, 'Invalid confidence for classifier.');
+    IBMNaturalLanguageClassifierV1Models.ClassifyOptionsBuilder ClassifyOptionsBuilder = classifyOptions.newBuilder();
+    ClassifyOptionsBuilder = new IBMNaturalLanguageClassifierV1Models.ClassifyOptionsBuilder(classifier_id_serialized_name, text_serialized_name);
+    Test.stopTest();
+  }
+  
+  /**
+   *  Test send data to create and train a classifier.
+   *
+   */
+  static testMethod void  testCreateClassifier(){
+    String body = IBMWatsonMockResponses.CreateClassifier();
+    IBMWatsonMockHttpResponse mockResponse = new IBMWatsonMockHttpResponse(200, 'Success', body, null);
+    Test.setMock(HttpCalloutMock.class, mockResponse);
+    Test.startTest();
+    IBMNaturalLanguageClassifierV1 service = new IBMNaturalLanguageClassifierV1();
+    service.setEndPoint('https://gateway.watsonplatform.net/discovery/api');
+    service.setUsernameAndPassword('username', 'password');
+    Attachment TrainingMetadata = new Attachment(Body=Blob.valueOf('test string'),
+      Name='test.txt',
+      Description='test description',
+      ContentType='text/plain');
+    IBMWatsonFile training_metadata_serialized_name = new IBMWatsonFile.FileBuilder()
+      .attachment(TrainingMetadata)
+      .build();
+    Attachment TrainingData = new Attachment(Body=Blob.valueOf('test string'),
+      Name='test.txt',
+      Description='test description',
+      ContentType='text/plain');
+    String training_metadata_filename_serialized_name = 'My Classifier Metadata';
+    IBMWatsonFile training_data_serialized_name = new IBMWatsonFile.FileBuilder()
+      .attachment(TrainingData)
+      .build();
+    String training_data_filename_serialized_name = 'My Classifier Data';
+    IBMNaturalLanguageClassifierV1Models.CreateClassifierOptions createClassifierOptions = 
+      new IBMNaturalLanguageClassifierV1Models.CreateClassifierOptionsBuilder()
+      .trainingMetadata(training_metadata_serialized_name)
+      .trainingMetadataFilename(training_metadata_filename_serialized_name)
+      .trainingData(training_data_serialized_name)
+      .trainingDataFilename(training_data_filename_serialized_name)
+      .build();
+    IBMNaturalLanguageClassifierV1Models.Classifier Classifier = service.createClassifier(createClassifierOptions);
+    System.assertEquals(Classifier.getName(),'My Classifier');
```

Comment:
leave space after `,`

Reply:
I think this should apply to all this file. I just saw multiple places where we don't leave an space